src/core/operator_list.js

```diff
@@ -106,15 +106,15 @@ var QueueOptimizer = (function QueueOptimizerClosure() {
 
       // assuming that heights of those image is too small (~1 pixel)
       // packing as much as possible by lines
-      var maxX = 0;
-      var map = [],
-        maxLineHeight = 0;
-      var currentX = IMAGE_PADDING,
+      let maxX = 0;
+      const map = [];
+      let maxLineHeight = 0;
+      let currentX = IMAGE_PADDING,
         currentY = IMAGE_PADDING;
-      var q;
+      let q;
```

Comment:
*Optional nit:* I believe that you can simply inline this definition in the loop headers below, since it doesn't seem to ever be accessed outside of the loops.